tensorflow/core/ops/ops.pbtxt

```diff
@@ -10313,6 +10313,154 @@ op {
   summary: "Computes gradients of the maxpooling function."
 }
 op {
+  name: "MaxPoolGradGrad"
```

Comment:
Thanks for making these changes, though you don't need to add them in this PR -- they are automatically generated later from the op registrations.